sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirPatchParameter.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.gcp.healthcare;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Objects;
+import org.apache.beam.sdk.coders.DefaultCoder;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+/**
+ * FhirPatchParemeter represents the parameters for a FHIR patch request, used as a parameter for
+ * {@link FhirIO.PatchResources}.
+ */
+@DefaultCoder(FhirPatchParameterCoder.class)
+public class FhirPatchParameter implements Serializable {
```

Comment:
it's not necessary, but you could replace this with an `@AutoValue` class that implements all the hashing / toString, etc. The AutoVAlue class can also have coders automatically set up.

https://github.com/google/auto/blob/master/value/userguide/index.md

Reply:
Done, thanks for tip!